sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -662,6 +662,16 @@ case class ShowCurrentNamespace(catalogManager: CatalogManager) extends LeafComm
     AttributeReference("namespace", StringType, nullable = false)())
 }
 
+/**
+ * The logical plan of the SHOW CATALOGS command.
+ */
+case class ShowCatalogs(
+    catalogManager: CatalogManager,
+    pattern: Option[String]) extends LeafCommand {
+  override val output: Seq[Attribute] = Seq(
```

Comment:
can we follow `ShowViews` and put outputs in an `object`? otherwise the output expr ID changes after plan copy.

Reply:
ok